main/ertp/guide/purses-and-payments.md

```diff
@@ -1,109 +1,107 @@
+Each `purse` and `payment` object contains a specific single amount of digital assets,
+which may be none at all ("empty" in [AmountMath](./amount-math.md) terms). In the same way
+you might have separate bank accounts for different purposes,
+you can have separate purses for the same `brand` of digital asset.
+One of your purses might hold 2 Quatloos while another holds 9000 Quatloos.
 
-When you deposit assets into a `purse` or `payment`, they are added to
-whatever assets already exist there. So a 3 Quatloos deposit 
-into a `purse` with 8 Quatloos, results in a `purse` with 11 Quatloos. Similarly, 
-then withdrawing 6 Quatloos from that `purse` into a new `payment` leaves the `purse` 
-with 5 Quatloos and the new `payment` has 6 Quatloos. 
+When you deposit assets into a `purse`, they are added to
+whatever assets already exist there. So a 3 Quatloos deposit
+into a `purse` with 8 Quatloos, results in a `purse` with 11 Quatloos. Similarly,
+then withdrawing 6 Quatloos from that `purse` into a new `payment` leaves the `purse`
+with 5 Quatloos and the new `payment` with 6 Quatloos.
 
 When adding a `payment` to a `purse`, you must add the entire `payment`. To
-only add part of a `payment`, you must first use `payment.split()` 
-or `payment.splitMany()` to split the `payment` into two or more new `payments`.
-
-`mints` create entirely new digital assets and put them in a new `payment`. 
-You also create a `payment` by withdrawing assets from a `purse`, by splitting an 
-existing `payment`, or by combining multiple `payments` into one new one. Note 
-the `brand` of the new `payment` is the same as the associated `brand` of its originating `mint`, `purse` or `payment`. 
-
-You don't transfer assets directly from one `purse` to
-another `purse`. Instead, in ERTP you do something like these steps to send and receive assets. The actual send operation is up to you; ERTP does not 
-implement a way of sending object-containing messages between parties.
-- Send assets:
+only add part of a `payment`, you must first use [`issuer.split()`](/ertp/api/issuer.md#issuer-split-payment-paymentamounta)
+or [`issuer.splitMany()`](/ertp/api/issuer.md#issuer-splitmany-payment-amountarray)
+to split it into two or more new `payments`.
+
+`mints` create entirely new digital assets and put them in a new `payment`.
+You also create a `payment` by withdrawing assets from a `purse`, by splitting an
+existing `payment`, or by combining multiple `payments` into one new one. Note
+the `brand` of the new `payment` is the same as the associated `brand` of its originating `mint`, `purse` or `payment`.
+
+In ERTP, assets are not transferred directly from one `purse` to another.
+Instead, the transfer must be mediated by a `payment` as demonstrated below.
+ERTP does not implement a way to perform the actual send and receive operations,
+although they are provided in the Agoric stack by [`E()`](/guides/js-programming/eventual-send.md).
```

Comment:
This seems confusing. Rather than point out that ERTP doesn't do it, just explain how to do it with `E`.